File: Leetcode-Problems/Furthest Point From Origin.cpp
class Solution
{
public:
    int furthestDistanceFromOrigin(string moves)
    {
        int ans = 0;
        int temp = 0;
        for (char c : moves)
        {
            if (c == 'L')
                ans--;
            if (c == 'R')
                ans++;
            if (c == '_')
                temp++;
        }
        return abs(ans) + temp;
    }
};

File: Leetcode-Problems/Furthest Point From Origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int ans=0,temp=0;
        for (char c : moves.toCharArray()) {
            if (c == 'L')
                ans--;
            else if (c == 'R')
                ans++;
            else temp++;
        }
        return Math.abs(ans)+temp;
    }
}